mountpoint-s3-crt-sys/Cargo.toml

```diff
@@ -24,7 +24,7 @@ exclude = [
     "crt/aws-lc/**/*test*.py",
     "crt/aws-lc/**/*test*.txt",
     # All the text files in kat/ directory are 'known answer tests'
-    "crt/aws-lc/crypto/ml_dsa/kat/*.txt",
+    "crt/aws-lc/crypto/fipsmodule/ml_dsa/kat/*.txt",
```

Comment:
To accomodate changes on https://github.com/aws/aws-lc/commit/fa44dbfb973177c1a93b52a33903a5edab889903